plugins/nodes/src/js/utils/compareValues.tsx

```diff
@@ -0,0 +1,21 @@
+// TODO: DCOS-39081
+export function compareValues(
+  a: string | null,
+  b: string | null,
+  aTieBreaker: string,
+  bTieBreaker: string
+) {
+  if (a === b || a == null || b == null) {
+    a = aTieBreaker;
+    b = bTieBreaker;
+  }
+
+  if (a > b) {
+    return 1;
+  }
+  if (a < b) {
+    return -1;
+  }
+
+  return 0;
```

Comment:
Does that really works for numbers represented as strings? and memory values. How does that sort 10Gb and 99Mb?

Also if it is only comparing Strings, I think it should state that on its name.

Reply:
Also for cpu, 10 cpus and 2 cpu. 

```js
"10" > "2"
false
```

Reply:
lets fix that later: https://jira.mesosphere.com/browse/DCOS-39112